docs/references.bib

```diff
@@ -1008,7 +1016,7 @@ @Book{            seligman1967
   mrreviewer    = {R. E. Block}
 }
 
-@book{            serre1965
+@Book{            serre1965,
```

Comment:
The missing comma is here (took me a bit of scrolling through the diff to find it).

Oddly enough, our website (which uses bibtool) has apparently been fine without the comma: https://leanprover-community.github.io/mathlib_docs/references.html#Ser87

Reply:
> The missing comma is here (took me a bit of scrolling through the diff to find it).

That's why I gave it its [own commit](https://github.com/leanprover-community/mathlib/pull/8585/commits/da10a20ff8024797099cb2188bda16fb195c4eb4)